types/office-js-preview/index.d.ts

```diff
@@ -42325,6 +42684,45 @@ declare namespace Excel {
          */
         hidden = "Hidden"
     }
+    /**
+     *
+     * Represents how the comment(s) in the event were changed.
```

Comment:
Showing the plural option with parentheses can be problematic for screen readers. I suggest just making it plural if this can be applied to multiple comments at once. Otherwise, just keep it singular. And use appropriate tense.